framework/src/wifi_manager/wifi_manager.c

```diff
@@ -448,7 +446,7 @@ wifi_manager_result_e wifi_manager_set_mode(wifi_manager_mode_e mode, wifi_manag
 		nvdbg("Wifi mode is changed to station");
 	}
 	else {
-			ndbg("Invalid config: %d %d %s\n", mode, config->channel, config->ssid);
+			ndbg("Invalid config!\n");
```

Comment:
There is no relationship between commit and changes in wifi_manager.c and wifi_utils.c. Please split commits.

Reply:
I can't split because wifi_manager uses apis of utils.